src/Orchard.Core.Tests/Common/Providers/CommonPartProviderTests.cs

```diff
@@ -215,11 +215,11 @@ class StubThemeService : IThemeManager {
 
             var user = contentManager.New<IUser>("User");
             _authn.Setup(x => x.GetAuthenticatedUser()).Returns(user);
-            _authz.Setup(x => x.TryCheckAccess(StandardPermissions.SiteOwner, user, item)).Returns(true);
+            _authz.Setup(x => x.TryCheckAccess(OwnerEditorPermissions.MayEditContentOwner, user, item)).Returns(true);
```

Comment:
Because the Owner editor was changed to check for this permission instead of SiteOwner.